int32 location_id
string ingredient_name
float32 desired_pickup_weight
---
float32 x
float32 y
float32 z